doc/source/performance/threadpool.rst

```diff
@@ -154,31 +154,45 @@ This variable can be used to define the number of threads that can use the CPU a
 
 The number of milliseconds before a running thread is considered stalled. When this limit is reached thread pool will wake up or create another thread. This is being used to prevent a long-running query from monopolizing the pool.
 
+.. rubric:: Upgrading from Percona Server for MySQL 8.0.14 to any higher version
```

Comment:
Are you sure about the title? It says upgrading from 8.0.14, while the description below says that 8.0.14 already removed these variables. Isn't this about upgrading from a version before 8.0.14 to 8.0.14+?